tests/vcs/git/GitCommitBearTest.py

```diff
@@ -87,62 +87,73 @@ def test_git_failure(self):
         git_error = self.msg_queue.get().message
         self.assertEqual(git_error[:4], 'git:')
 
-        self.assert_no_msgs()
-
     def test_empty_message(self):
         self.git_commit('')
 
         self.assertEqual(self.run_uut(),
-                         ['HEAD commit has no message.'])
+                         ['HEAD commit information',
```

Comment:
We probably might want to modify `self.run_uut()` to just ignore the `CommitResult` in this test, so we don't need to modify these testcases
And `GitCommitBear` is a bear that's used in CLI, which means it just shows the `Result` to the CLI, rather than `HiddenResult`